eCommerce.Products.Api/DependencyInjections/DependencyInjectionExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using eCommerce.Products.Core.Products.Catalag.Models;
+using eCommerce.Products.Core.Products.Catalag.Repositories;
+using eCommerce.Products.Core.Products.Catalag.Services;
+using eCommerce.Products.Core.Products.Catalag.Services.Impl;
+using eCommerce.Products.Core.Products.Catalag.Validators;
+using eCommerce.Products.Infrastructure.Repositories;
+using FluentValidation;
+
+namespace eCommerce.Products.Api.DependencyInjections
```

Comment:
We are going to change this folder name, Please replace the folder name DependencyInjections by Extensions